linear-algebra/source/future-ON/01.ptx

```diff
@@ -111,7 +111,42 @@ The dot product combines two vectors and creates a scalar that gives us geometri
         <p>Two vectors <m>\vec u</m> and <m>\vec v</m> in <m>\IR^n</m> are <term>orthogonal</term><idx>orthogonal</idx> provided <m>\vec u \cdot \vec v = 0</m>.
         </p></statement></definition>
 
-<p>A bunch of stuff goes here. So it turns out you can find the angle between vectors using the Law of Cosines as a starting place.</p>   
+<definition>
+    <statement>
+        <p> Given two vectors <m>\vec u</m> and <m>\vec v</m> in <m>\IR^n</m>, the <term>distance</term> between the vectors, denoted <m>d(\vec u,\vec v)</m> is given by
+            <me>
+                d(\vec u,\vec v)=\lvert \vec u-\vec v\rvert.
+            </me>
+        </p></statement></definition>
+
+
+<activity estimated-time="5">
+    <introduction>
+        <p>
+            Consider the vectors <m>\vec u=\left[\begin{array}{c} 2 \\ 1  \end{array}\right]</m> and <m>\vec v=\left[\begin{array}{c} 1 \\ 3  \end{array}\right].</m>
+        </p>
+    </introduction>
```

Comment:
This activity (and the ones you didn't write before it) could use a bit more 4S scaffolding. My gut instinct is that at this stage, task A should be replaced with a given illustration to work off of. I think practicing drawing these sorts of pictures would work better as fluency builders or flexible extension.